tests/test_observatory.py

```diff
@@ -84,3 +85,21 @@ def test_grid_baselines(bm):
     grid0 = a.grid_baselines()
     grid1 = a.grid_baselines(bl_coords, bl_counts)
     assert np.allclose(grid0, grid1)
+
+
+def test_min_max_antpos(bm):
+    a = Observatory(
+        antpos=np.array([np.linsapce(0, 50, 11), np.zeros(11), np.zeros(11)]),
+        beam=bm,
+        min_antpos=7,
+    )
+
+    assert len(a.antpos) == 9
+
+    a = Observatory(
+        antpos=np.array([np.linsapce(0, 50, 11), np.zeros(11), np.zeros(11)]),
```

Comment:
some easy typos here causing test failures

Reply:
oh good. OK that's fixed.